include/mbedtls/pem.h

```diff
@@ -103,6 +103,23 @@ int mbedtls_pem_read_buffer( mbedtls_pem_context *ctx, const char *header, const
                      const unsigned char *pwd,
                      size_t pwdlen, size_t *use_len );
 
+/**
+ * \brief       Retrieve pointer to der data stored in \c mbedtls_pem_context.
+ *
+ * \param ctx       context to use
+ * \param derlen    destination for length of der data
+ *
+ * \return          const unsigned char * pointer to der data stored in
+ *                  \c mbedtls_pem_context.  Pointer is valid while
+ *                  \c mbedtls_pem_context is not further modified or freed.
+ */
+static inline const unsigned char *mbedtls_pem_get_der( mbedtls_pem_context *ctx, size_t *derlen )
```

Comment:
Note: I know I said I like that name before, but now I'm having second thoughts: technically speaking, the decoded data doesn't have to be DER, it could also be BER. Perhaps we could go with something more generic such as "decoded data" or "binary data" or just "data"? OTOH, colloquially everyone says "DER" to mean the binary version as opposed to the base64/pem-encoded version, so `_der` makes sense.

Reply:
I see elsewhere the use of `*_raw`.  That would work, too, instead of `*_der`.  Do you like that?

Reply:
Sounds good too. But let's not make any change until the second reviewer weighs in. It's not that I dislike `_der`, I'm just unsure.

Reply:
ackn